<launch>
  <group ns="aero/teleop">  
    <node pkg="joy" type="joy_node" name="joy_node" output="screen">
      <param name="dev" value="/dev/input/js0" />
    </node>
    <node pkg="aero_teleop" type="teleop" name="aero_teleop" output="screen">
      <param name="turn_scale" value="1.2" />
      <param name="drive_scale" value="1.5" />
      <param name="deadman_button" value="1" />
    </node>
  </group>
</launch>
